src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml

```diff
@@ -26,6 +26,12 @@
             <tag name="serializer.normalizer" priority="-1000" />
         </service>
 
+        <!-- Json Normalizer -->
+        <service id="serializer.normalizer.json" class="Symfony\Component\Serializer\Normalizer\JsonNormalizer" public="false">
```

Comment:
Should be `Symfony\Component\Serializer\Normalizer\JsonSerializableNormalizer` or even better: rename the class to `JsonNormalizer`